src/content/2025-04-10-osc-at-wicse-codeathon.md

```diff
@@ -0,0 +1,26 @@
+---
+slug: "/2025-wicse-codeathon"
+date: "2025-04-10"
+title: "OSC at WICSE Code-a-thon"
+author: ["Nicolas Valiente"]
+featuredImage: ../images/blog/2025-04-10-wicse-codeathon-workshop.jpeg
+subtitle: Educating the youth to become future FAANG front-end developers
+---
+
+On April 29th, 2025, the Women in Computer Science and Engineering (WICSE) held their annual Code-a-thon. This event marks the day of the year where middle and high schoolers are invited onto UF campus to be exposed to a variety of computing topics. From quantum computing to cybersecurity, there was no shortage of topics to learn. To support WICSE's endeavor and to provide the kids with an incredible experience, high-profile OSC members enlisted to attend and host the most essential workshop...
```

Comment:
You should massively over exaggerate the difficulty of the code a thon, that way the juxtaposition makes our achievements look even greater.

Seeing other workshop groups in disarray, panic spread among the volunteers. The nearer the time came, the more nervous everyone grew. Among the WiCSE board and the ranks of volunteers, there were some who expressed the opinion that it was not in WiCSE’s own interest to lead workshops of such great scale and magnitude. Rumours were exchanged among participants, and these filled the volunteers with dread. Anton Salvador, president of OSC, intervened on behalf of the volunteers. He reassured WiCSE that the breadth of OSC’s knowledge and experiences was not to be doubted and made concessions for the workshops to continue… etc.